### DISCUSSION TITLE
word/pkpm/PKPM%E8%AE%BE%E7%BD%AE

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#532] [Announcements]
# word/pkpm/PKPM%E8%AE%BE%E7%BD%AE

[TOC] 注意事项 地库子项： 1.消防车荷载是否组合：可以通过点选消防车荷载处的梁板配筋查看工况。 2.桩刚度 3.基床系数 4.高层基础是否勾选楼层荷载折减。 5.抗浮水位。 6.基础板面荷载中的恒活，地库外墙范围外的筏板覆土恒载。 7.基础模型中的筏板混凝土等级。 其他地上部分子项： 1.悬挑梁，梁布置修改等处的点铰复查。 2.修改楼层组装或其他类似修改后，核查混凝土等级。 3.楼层信息里的钢筋等级。

http://localhost:8080/word/pkpm/PKPM%E8%AE%BE%E7%BD%AE.html

**Comment by yy0327** [comment]
11
